dcos/mesos.py

```diff
@@ -30,14 +30,10 @@ class DCOSClient(object):
 
     def __init__(self):
         config = util.get_config()
```

Comment:
My bad: I dummied and forgot to follow config to realize that it would not have the function. I changed out the config in line 35 with util to fix it and `config` is not used anywhere else inside the class (aside form the __init__).